FlaskApp/api/utils.py

```diff
@@ -0,0 +1,70 @@
+import re
+class Validator():
+    def __init__(self):
+        self.ride_ofer_props = ["name","From","To","car_model","cost","seats_available", "time", ]
+        self.ride_request_props = ["name", "From", "To", "seats_needed", "time"]
+        self.register_props = ["name","username","email", "password"]
+        self.has_numbers = re.compile('[0-9]')
+        self.has_special = re.compile('[^\w\s]')
+
+    def validate(self, obj, activity):
+        if activity == 'create_ride':
+            for prop in self.ride_ofer_props:
+                if prop not in obj:
```

Comment:
You could split your validators into different functions instead of using an if-else statement

Reply:
Noted